src/content/footer-info.json

```diff
@@ -0,0 +1,15 @@
+{
+	"items": [{
+			"link": "https://www.facebook.com/cutc2021/",
+			"image": "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAEAAAABACAQAAAAAYLlVAAAABGdBTUEAALGPC/xhBQAAACBjSFJNAAB6JgAAgIQAAPoAAACA6AAAdTAAAOpgAAA6mAAAF3CculE8AAAAAmJLR0QAAKqNIzIAAAAJcEhZcwAADsQAAA7EAZUrDhsAAAAHdElNRQfkARUGCBtdl0J/AAADqElEQVRo3sWZTWxUVRiGn+92aDCUOm1qJcZEES1gdVFDNy5KXOBOgwujqAipGncSSSWoCf6sVASamBiJC8KCqnHThXbjyhSMiRpNhiI1/CQmJtIyDJVqUirzuqjDzL1z5879nXlXc+/5znne83/nHCOi1M8Ig2xigB7ydAGLXKXELGeZYdrmopYYHj2swyqorCCVVdAhbUkb3a0xnVEUzWhMa9OB9+odXYkEr+iK3lZPMrjpec3FgldU1B45cfH36lQieEUntSEO/gmVUsFL0l96Ohrc0ZHU4BUdCt0V6tRE6nhJOqFV4fBfZ4KXpK+aWpDpeGZ4SZpo0hEZ9L1XB4PwT2aOl6QdtUyrwW/gJ26NNlljaYGH7EKdARkneThx4eIMv1NErKaTNfRwO3fWRU2z1eTN+ULipv1Vo7rNx5OfdnmDejWfCP6v9jUa377xl5R3B72bEP94QK/460BtSHfMDbeifYHDwl/Fmu8FjSXCz6ojhgFpbzUk2teOV6NNJkYjFSoBw4nwZb+RH8qANAQ5YAdJVLB5D3A1r/MYd9DZNO8z/JwDtiUycLHuzac8FzLvNnDUz2AiA8W6+of/+nlQfQ4jtftBDF3zPK8jFzqvw4jDA4nw4F3T10TKPeiwMaGBZNrocF9bDQw49LXVQJ9DOv/g4mqtaSnEguHWlGueL9k/tYnqoNsV/SifB5S1FH7KVHXdSo0T7QauVAV3sTksxrAQResDU/926haStHV3YOqiw+W2tsAfDr9lbOCewNSLDrNZ0pUnHxhwzuF0pvVf3yT9B1M/f0bcD5ddM+cTe8NV5018V/OYa7LQrcvZnGYi7oirqD16usWT2kH4g6mCXXKAbyLh09QUOMBE2wxMggP2Y8YDsZHO2vcrLQDH22LgKFQMHKWUqKg4usqxmwbsGh+13MC4LVRbAMaZT1BYdM0xvvLjfwNWYn9LDexfqX+1BeAYp1qGn64O+5sGTOxioSX4RUatXGcA7DwvtcTAy3au+uA617EvOZw5/j1zrbzeg6XXMl6Yv+BN9wuPASuzm6nM8JPsrPa+fwtgy2znRCb4z3jKlr0vfc72bJmdfJg6/n2etesR4rU95NHdEU++QZ+YgCubhqf3NskWplOp+7cMWcO/ZwHXB3aBrewm2VXsZV7kETufoATldUDFWF2woLfSuj3t0l6djmRgVq+oOx6tsY0hHdQvutHEwF36QMPhS418QqY+RrifzQzQS54uPrZXk1TrPwwITs/0xP6NAAAAJXRFWHRkYXRlOmNyZWF0ZQAyMDIwLTAxLTIxVDA2OjA4OjI3KzAwOjAwlZpBJgAAACV0RVh0ZGF0ZTptb2RpZnkAMjAyMC0wMS0yMVQwNjowODoyNyswMDowMOTH+ZoAAAAZdEVYdFNvZnR3YXJlAHd3dy5pbmtzY2FwZS5vcmeb7jwaAAAAAElFTkSuQmCC"
```

Comment:
Woah, is this what you got when you copied from Figma?? Wild (Base64-encoded image is not what I expected). I would recommend this file format:

For the images: Store in "assets>socials>{socialMediaExample.svg}"
Preferably copy them as SVGs, not base 64.

Instead of a JSON, store references to the images & links in the data section of the JS Vue object. Then use the for loop like you did.

```
data() {
    return {
      items: [
        {"link": "link.com", "image": "socialMediaExample.svg"}
      ]
    };
  }
```

Reply:
Everything inside the svg tag and including it. This is what I got when I copied as svg